### PR TITLE
Fix us02 s03 instructions modal

### PR DESCRIPTION
**Pull request by sorrento974**

### CODE REVIEW

**Review by Alice-Berthelot — APPROVED** [review]

**Review by Dinosaure7 — APPROVED** [review]
Looks all good to me
Do not push json from root

File: client/src/components/ModalContent.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { PropTypes } from "prop-types";
+
+function ModalContent({ onClose }) {
+  return (
+    <section className="modal">
```
Comment (Yoann-Fortin) [inline client/src/components/ModalContent.jsx:5]: Dialog tag is require here

File: client/src/components/ModalContent.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      <button type="button" className="close-button" onClick={onClose}>
```
Comment (Yoann-Fortin) [inline client/src/components/ModalContent.jsx]: Aria label is missing

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: client/src/components/ModalContent.jsx (hunk 2 above)
Comment (Dinosaure7) [reply client/src/components/ModalContent.jsx]: done

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**